lib/ddtrace/monkey.rb

```diff
@@ -16,8 +16,10 @@ module Monkey
     #   can be call twice but should just do nothing the second time.
     # - patched?, which returns true if the module has been succesfully
     #   patched (patching might have failed if requirements were not here)
-    @patchers = { elasticsearch: Datadog::Contrib::Elasticsearch::Patcher,
-                  redis: Datadog::Contrib::Redis::Patcher }
+    @patchers = {
+      elasticsearch: Datadog::Contrib::Elasticsearch::Patcher,
+      redis: Datadog::Contrib::Redis::Patcher
```

Comment:
Mmm, IMHO it makes sense if it's enabled for "all Sinatra instances". Like, if after doing "something", any Sinatra app you create has it enabled. Sort of "class-level patch". What you describe here is more like a per-instance setting, therefore it probably does not belong to the `get_patched_modules` list - which is really about "altered classes", and does not refer to the hackiness ;)